Makefile

```diff
@@ -25,9 +25,8 @@ REPO_GIT_NAME ?= $(shell git config --get remote.origin.url)
 
 ITEMS       ?= 1 2
 IMAGE_TYPES ?= metanorma mn
-VERSIONS		?= 1.1.8 1.1.8
-ROOT_IMAGES ?= ubuntu:18.04 ubuntu:18.04
-RUBY_VER = 2.5.4
+VERSIONS		?= 1.1.8 1.2.2
```

Comment:
I suppose both versions should be 1.2.2 here

Reply:
Actually, I'm out of the context. I didn't have a chance to get into `metanorma-docker` yet.

Reply:
> I suppose both versions should be 1.2.2 here

Yes, @CAMOBAP795 can you make a PR? Thanks.

Reply:
@andrew2net sorry I tagged you wrongly 🤣